spec/lib/core/entity/article_spec.rb

```diff
@@ -224,6 +224,69 @@ module Core
       end
     end
 
+    describe '#related_links' do
+      let(:related_content) do
+        {
+          'related_links' => [
+            {
+              'title' => 'Most related link.',
+              'path' => '/most-related-link'
+            },
+            {
+              'title' => 'Next related link.',
+              'path' => '/next-related-link'
+            }
+          ]
+        }
+      end
+
+      context 'provide data' do
+        it 'has 2 article links' do
+          expect(subject.related_links.length).to eq(2)
+          expect(subject.related_links.first).to be_an_instance_of(ArticleLink)
```

Comment:
This and the following assertions on type are no longer necessary now the interface of this is specified elsewhere, I guess these are a lay over from your TDDing?